lib/core/src/sdk.rs

```diff
@@ -95,11 +101,39 @@ impl LiquidSdk {
         Ok(sdk)
     }
 
+    fn validate_api_key(api_key: &str) -> Result<()> {
+        let pem = [
+            "-----BEGIN CERTIFICATE-----",
+            api_key,
+            "-----END CERTIFICATE-----",
+        ]
+        .join("\n");
```

Comment:
Alternatively, we can use `include_bytes!` macro and pass the file path to the cert file instead (perhaps send an email which downloads the certificate directly).

Reply:
I would say most people are injecting the API key as an env variable and don't necessarily have easy access to the file system